Explains how to send product/purchase metadata from Shopify for analysis (#6275)
PostHog/posthog.com: contents/docs/libraries/shopify.md

<input type="submit" class="add-to-cart-button" value="{{ 'products.product.add_to_cart' | t }}" 
    data-ph-capture-attribute-product-name="{{product.title}}" 
    data-ph-capture-attribute-product-price="{{product.price | divided_by: 100 }}"
/>
```

In the above example, anything prefixed with `data-ph-capture-attribute` will be sent to PostHog as a property using the value that appears in the DOM on the page.

Now when viewing an event, you'll see this information attached to the autocapture event where the _Add to cart_ button is clicked.

![Data attributes in an autocaptured event](../../images/tutorials/shopify/autocapture-data-attributes.png)

### 3. Order details (using properties on a custom event)

> This requires a Shopify Plus account because you'll need to modify your store's checkout settings. This option is available in the Shopify admin by navigating to _Settings_ → _Checkout_ → _Order status page_ → _Additional scripts_.

You can send any info you want to PostHog by [attaching properties to a custom event](/docs/getting-started/send-events#sending-custom-properties-on-an-event). In this example, we'll send information about the order that was just placed.

> > This code should be used as a guide and may need to be modified depending on the setup of your store.

Reviewer: ```suggestion
```

```html
{% if first_time_accessed %}
<script>
    posthog.capture('order-submitted', {
        value: '{{ order.total_price }}', 
        created_at: '{{ order.created_at }}', 
        order_number: '{{ order.order_number }}',
        userId: '{{ customer.id }}',
        orderId: '{{ checkout.order_id }}',
        "products": [{% for line_item in line_items %}{
            "productId": '{{ line_item.product.type }}',
            "colorId": '{{ line_item.variant.option1 }}',
            "size": '{{ line_item.variant.option2 }}',
            "quantity": {{ line_item.quantity }},
            "price": {{ line_item.final_price| divided_by: 100.0 }},
            "currency": '{{ checkout.currency }}',
            "sku": '{{ line_item.product_id }}_{{ line_item.variant_id }}',
            }{% unless forloop.last %}, {% endunless %}{% endfor %}]
    });
</script>
{% endif %}
```

> **Tip:** Be sure to wrap the code in `{% if first_time_accessed %}` to ensure the event is only sent once.

> Below you'll see the order's total price including taxes (`value`) and the array of products that were ordered with some specific info about the variants:

Reviewer: ```suggestion
Once setup, you can see the order's total price including taxes (`value`) and the array of products that were ordered with specific info about the variants in PostHog:
```
